### DISCUSSION TITLE
Release v0.1.20

### DISCUSSION THREAD

**Discussion by ehendrix23** [ehendrix23/tesla_dashcam#186] [Announcements]
- BREAKING CHANGE:
    - How offsets are calculated has been changed and can impact result if negative values were being provided for start_offset and/or end_offset!
    
- New:
    - Option --sentry_start_offset to set the starting offset specifically for sentry based events
    - Option --sentry_end_offset to set the ending offset specifically for sentry based events
    - Added support for Intel VAAPI GPU acceleration (https://trac.ffmpeg.org/wiki/Hardware/VAAPI). Contributed by timrettop.
    - Two new dockerfiles have been added. One to support NVIDIA and other one to support VAAPI. Contributed by magicalyak

- Changed:
    - Updated Docketfile. Contributed by magicalyak
 
- Fixed:
    - Resolved issue on Linux with default device initialization failure when using Intel GPU acceleration and multiple usable devices by adding new qsv gpu_type (https://trac.ffmpeg.org/ticket/7649). Contributed by timrettop.

<hr /><em>This discussion was created from the release <a href='https://github.com/ehendrix23/tesla_dashcam/releases/tag/v0.1.20'>Release v0.1.20</a>.</em>

**Comment by bsanborn** [comment]
Thanks for all your work on this. Is there something we have to do to get this update for docker?
